frontend/src/metabase/admin/permissions/components/LegacyPermissionsModal/LegacyPermissionsModal.tsx

```diff
@@ -0,0 +1,66 @@
+import { Link } from "react-router";
+import { t } from "ttag";
+
+import { useModalOpen } from "metabase/hooks/use-modal-open";
+import { useSelector } from "metabase/lib/redux";
+import { getDocsUrl } from "metabase/selectors/settings";
+import { Button, Flex, Modal, Text } from "metabase/ui";
+
+export const LegacyPermissionsModal = ({
+  isOpen,
+  onClose,
+}: {
+  isOpen: boolean;
+  onClose: () => void;
+}) => {
+  //Used to animate the modal
+  const { open: showModal } = useModalOpen();
+
+  const docsUrl = useSelector(state =>
+    getDocsUrl(state, { page: "permissions/no-self-service-deprecation" }),
+  );
+  return (
+    <Modal.Root
+      opened={isOpen && showModal}
+      onClose={onClose}
+      size="35rem"
+      closeOnClickOutside={false}
+    >
+      <Modal.Overlay />
+      <Modal.Content>
+        <Modal.Header p="2.5rem" pb="1.5rem">
+          <Modal.Title>
+            <Text fz="1.25rem">
+              {t`Your data permissions may look different, but the access hasn’t changed.`}
+            </Text>
+          </Modal.Title>
+        </Modal.Header>
+        <Modal.Body p="2.5rem">
+          <Text mb="1rem">
+            {t`In Metabase 50, we split our data permissions into two new settings:`}{" "}
+            <Text span color="brand" fw="bold">{t`View data`}</Text> {t`and`}{" "}
+            <Text span color="brand" fw="bold">{t`Create queries`}</Text>
+            {`. Separate settings for what people can view and what they can query makes data permissions more expressive and easier to reason about. `}
```

Comment:
I think you want to use the ttag method that allows you to use jsx. so that the first part of this string (line 40) and the last part (line 43) aren't separate for the translators.

(Also complete side note, but do we ever update the translations for a release? Won't this always be in english for v50?)